### PR TITLE
BZ1881864: [Docs] [Prereq] [Admin]  Image I/O Proxy replaced by ovirt-imageio

### PR DESCRIPTION
**Pull request by stoobie**
Fixes BZ1881864: https://bugzilla.redhat.com/show_bug.cgi?id=1881864

Changes proposed in this pull request:

- Changing *ImageIO Proxy server* to *ovirt-imageio*

Previews:
https://ovirt.github.io/ovirt-site/previews/2682/documentation/installing_ovirt_as_a_self-hosted_engine_using_the_command_line/index.html#RHV-manager-firewall-requirements_SHE_cli_deploy
https://ovirt.github.io/ovirt-site/previews/2682/documentation/installing_ovirt_as_a_self-hosted_engine_using_the_command_line/index.html#host-firewall-requirements_SHE_cli_deploy
https://ovirt.github.io/ovirt-site/previews/2682/documentation/administration_guide/index.html#Uploading_Images_to_a_Data_Storage_Domain_storage_tasks

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @stoobie

This pull request needs review by: @nirs

### CODE REVIEW

File: source/documentation/common/prereqs/ref-Host_Firewall_Requirements.adoc [hunk 1]
```diff
@@ -5,7 +5,7 @@
 // PPG
 // Install
 
-{enterprise-linux-host-fullname}s and {hypervisor-fullname}s ({hypervisor-shortname}) require a number of ports to be opened to allow network traffic through the system's firewall. The firewall rules are automatically configured by default when adding a new host to the {engine-name}, overwriting any pre-existing firewall configuration. 
+{enterprise-linux-host-fullname}s and {hypervisor-fullname}s ({hypervisor-shortname}) require a number of ports to be opened to allow network traffic through the system's firewall. The firewall rules are automatically configured by default when adding a new host to the {engine-name}, overwriting any pre-existing firewall configuration.
```
Comment (nirs) [inline source/documentation/common/prereqs/ref-Host_Firewall_Requirements.adoc:9]: Good change, but related to this PR - it is best to submit such cleanups separately.

File: source/documentation/common/prereqs/ref-Host_Firewall_Requirements.adoc [hunk 2]
```diff
@@ -81,9 +81,9 @@ Client machines |{hypervisor-fullname}s
 
 {enterprise-linux-host-fullname}s |VDSM communications with the {engine-name} and other virtualization hosts.
 |Yes
-|H11 |54322 |TCP |{virt-product-fullname} {engine-name} (ImageIO Proxy server) |{hypervisor-fullname}s
+|H11 |54322 |TCP |{virt-product-fullname} {engine-name} `ovirt-imageio` service |{hypervisor-fullname}s
 
-{enterprise-linux-host-fullname}s |Required for communication with the ImageIO daemon (*ovirt-imageio-daemon*).
+{enterprise-linux-host-fullname}s |Required for communication with the `ovirt-imageo` service.
```
Comment (nirs) [inline source/documentation/common/prereqs/ref-Host_Firewall_Requirements.adoc:86]: Looks good

File: source/documentation/common/prereqs/ref-Manager_Firewall_Requirements.adoc [hunk 3]
```diff
@@ -51,7 +51,7 @@ VM Portal clients |{virt-product-fullname} {engine-name} |Provides websocket pro
 
 {enterprise-linux-host-fullname}s |{virt-product-fullname} {engine-name} |If Kdump is enabled on the hosts, open this port for the fence_kdump listener on the {engine-name}. See link:{URL_virt_product_docs}{URL_format}administration_guide/index#sect-fence_kdump_Advanced_Configuration[fence_kdump Advanced Configuration]. `fence_kdump` doesn't provide a way to encrypt the connection. However, you can manually configure this port to block access from hosts that are not eligible.
 |No
-|M7 |54323 |TCP |Administration Portal clients |{virt-product-fullname} {engine-name} (ImageIO Proxy server) |Required for communication with the ImageIO Proxy (`ovirt-imageio-proxy`).
+|M7 |54323 |TCP |Administration Portal clients |{virt-product-fullname} {engine-name} (`ovirt-imageio` service) |Required for communication with the `ovirt-imageo` service.
```
Comment (nirs) [inline source/documentation/common/prereqs/ref-Manager_Firewall_Requirements.adoc:54]: Looks good.

**Review by nirs — APPROVED** [review]

**Review by dcdacosta — APPROVED** [review]
LGTM!

**Pull request merged**